cdhweb/pages/management/commands/exodus.py

```diff
@@ -429,3 +449,19 @@ def image_exodus(self):
                     # seems to mean that height/width calculation failed
                     # (usually non-images)
                     logging.warning("%s: %s" % (imgpath, err))
+
+    nav_order = [
+        # landing pages in order
+        'people', 'projects', 'research', 'engage', 'events', 'updates',
+        # top-level content pages in order
+        'about', 'annual-report', 'history', 'contact', 'credits'
+    ]
+
+    def adjust_toplevel_order(self, homepage):
+        # find and move each child page to first child
+        # in reverse of desired order; any pages not in our
+        # list should be ignored and end up at the end
+        for slug in reversed(self.nav_order):
+            page_to_move = homepage.get_children().get(slug=slug)
+            # make this page the first child
+            page_to_move.move(homepage, 'first-child')
```

Comment:
i think `.move()` is an event that is stored in the wagtail audit log, so for consistency with stuff i did in the history PR/story you might want `.move(log_action=False)` to prevent this showing up in the history

Reply:
ok, thanks for flagging